src/openzaak/config/templates/config/config_internal.html

```diff
@@ -0,0 +1,54 @@
+{% extends "master.html" %}
+{% load i18n sniplates %}
+
+{% block content %}
+
+<div class="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+    <h1 class="display-4">Service configuratie: NLX</h1>
+    <p class="lead">Step 2 of 2</p>
+</div>
+
+<div class="container">
+    <form method="post" enctype="multipart/form-data">
+        {% csrf_token %}
+        {% load_widgets form='snippets/forms.html' %}
+
+        {{ formset.management_form }}
+        {{ formset.non_form_errors }}
+
+        <div class="form-group row">
+            <div class="col"><strong>Service</strong></div>
+            <div class="col text-center"><strong>Enabled?</strong></div>
```

Comment:
This needs some sort of info icon with a popover with additional information explaining the meaning of this setting.

Reply:
tooltip is added